db/deploy/computed_columns/application_status@1.11.0.sql

```diff
@@ -0,0 +1,17 @@
+-- Deploy ccbc:computed_columns/application_status to pg
+-- requires: tables/application_status
+
+begin;
+
+create or replace function ccbc_public.application_status(application ccbc_public.application) returns varchar as
+$$
+select status from ccbc_public.application_status
+    where application_id= application.id
+    order by id desc limit 1;
+$$ language sql stable;
+
+grant execute on function ccbc_public.application_status to public;
+
+comment on function ccbc_public.application_status is 'Computed column to return status of an application';
+
+commit;
```

Comment:
When you do a `sqitch rework` this file tagged `@1.11.0` is supposed to be the same as the original file and any changes happen in the original `computed_columns/application_status` file. 

Yes it's confusing and I did the same thing before. I think the tag is to let us know which release our app was on as a history for the file.

You can see this by looking at other @ tagged files in the project, they are older versions of the original.

Reply:
hmm.. I understand it as `changes were made to the application after it was released with version 1.11.0 ` .
So I believe name is correct

Reply:
```Commit: 9f6bbbc8c058e721e2b5de6a529baff564d034b9 [9f6bbbc]
Parents: 6026d29853
Author: marcellmueller <marcel@button.is>
Date: October 26, 2022 12:55:07 PM
Committer: marcellmueller
chore: release v1.11.0

Reply:
The issue isn't the name/release version, it's that the changes have to happen in the original `application_status` file as you can see by looking at other tagged release files (their contents are older).